repos/fdbt-site/src/utils/index.ts

```diff
@@ -220,3 +220,11 @@ export const checkIfMultipleOperators = (ctx: NextPageContextWithSession): boole
 
 export const getCsrfToken = (ctx: DocumentContextWithSession | NextPageContextWithSession | NextPageContext): string =>
     (ctx.res as ResponseWithLocals)?.locals?.csrfToken ?? '';
+
+export const toArray = (thing: string | string[] | undefined): string[] => {
```

Comment:
I think this is repeated in `repos/fdbt-site/src/pages/api/salesOfferPackages.ts` - shall we import from utils in this location too?

Reply:
Good idea 👍